src/table.ts

```diff
@@ -719,34 +715,9 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
    * region_tag:bigtable_api_table_readstream
    */
   createReadStream(opts?: GetRowsOptions) {
-    const options = opts || {};
+    const options: GetRowsOptions = opts || {};
     const maxRetries = is.number(this.maxRetries) ? this.maxRetries! : 10;
```

Comment:
Should this 10 be defined as a default somewhere else so it's easier to keep track of and debug?

Reply:
I made a change to address this.